test/jdk/com/sun/tools/attach/java.policy.allow

```diff
@@ -15,7 +13,8 @@ grant {
     permission java.lang.RuntimePermission "loadLibrary.attach";
     permission java.util.PropertyPermission "sun.jvmstat.*", "read";
 
-    /* to read configuration file in META-INF/services, and write/delete .attach_pid<pid> */
-    permission java.io.FilePermission "<<ALL FILES>>", "read,write,delete";
+    /* to read configuration file in META-INF/services, write/delete .attach_pid<pid>, 
+    and read symbolic link of /proc/self/ns/mnt */
```

Comment:
```suggestion
    /* 
     * To read configuration file in META-INF/services, write/delete .attach_pid<pid>, 
     * and read symbolic link of /proc/self/ns/mnt.
     */
```

Reply:
Thanks for the review. The comment has been changed according your advice.